src/main/java/net/viktorc/pp4j/impl/JavaProcessPoolExecutor.java

```diff
@@ -288,6 +289,8 @@ public ProcessManager newProcessManager() {
 				for (URL url : urlClassLoader.getURLs()) {
 					try {
 						classPathEntries.add(Paths.get(url.toURI()).toAbsolutePath().toString());
+					} catch (FileSystemNotFoundException e){

```

Comment:
Could be combined into a single catch clause i.d.:
`catch (FileSystemNotFoundException | URISyntaxException  e)`